skxray/cdi.py

```diff
@@ -281,6 +283,10 @@ def cdi_recon(diffracted_pattern, sample_obj, sup,
     n_iterations : int, optional
         number of iterations to run.
         default is 1000.
+    plot_function : function, optional
+        plotting function
```

Comment:
Note that this is a callback function that expects to receive these four objects: `sample_obj, obj_error, diff_error, sup_error`